scripts/dockerfiles/build.dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM public.ecr.aws/amazonlinux/amazonlinux:2
+RUN yum update -y && yum install -y golang make tar rpm-build
```

Comment:
I think the version of golang should probably be fixed?

Reply:
I like that idea, but since the [spec file](https://github.com/aws/amazon-ecs-init/blob/master/packaging/amazon-linux-ami/ecs-init.spec#L52) only specifies a minimum, I _think_ that tracking the latest installed version is closer to what actually happens in authoritative builds.